### PR TITLE
Get all my loans

### PR DESCRIPTION
**Pull request by saman1357**
add getUserData feature (Front and Backend) with Tests and some code smell fixes

### CODE REVIEW

File: backend/src/main/java/de/neuefische/capstone/backend/repositories/MyLoansRepository.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package de.neuefische.capstone.backend.repositories;
+
+import de.neuefische.capstone.backend.models.UserData;
+import org.springframework.data.mongodb.repository.MongoRepository;
+@org.springframework.stereotype.Repository
```
Comment (cananozyol) [inline backend/src/main/java/de/neuefische/capstone/backend/repositories/MyLoansRepository.java:5]: import org.springframework.stereotype.Repository;

File: backend/src/main/java/de/neuefische/capstone/backend/services/MyLoansService.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+@RequiredArgsConstructor
+public class MyLoansService {
+    private final MyLoansRepository myLoansRepository;
+    String userId="0001";
```
Comment (cananozyol) [inline backend/src/main/java/de/neuefische/capstone/backend/services/MyLoansService.java:14]: maybe a UUId service is a option here?

**Review by cananozyol — APPROVED** [review]
really god job saman!! a huge first vertical slice!
LGTM! 🚀

File: backend/src/main/java/de/neuefische/capstone/backend/services/MyLoansService.java (hunk 2 above)
Comment (saman1357) [reply backend/src/main/java/de/neuefische/capstone/backend/services/MyLoansService.java:14]: definitely. I will do it later...

**Pull request merged**